releasenotes/notes/move_propagation.utils_to_trace_utils-6bdfed9fcb98a995.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    The ``propagation.utils`` module has been removed. Public methods have been moved to ``trace_utils.py``
```

Comment:
```suggestion
    The ``propagation.utils`` module has been removed. Public methods have been moved to ``ddtrace.contrib.trace_utils``
```

Reply:
we just want to state the module path that they were moved to, the file name won't help anyone.